admin-part/src/test-integration/kotlin/e2e/BuildMethodsTests.kt

```diff
@@ -0,0 +1,39 @@
+package com.epam.drill.plugins.coverage.e2e
+
+import com.epam.drill.builds.*
+import com.epam.drill.e2e.*
+import com.epam.drill.plugins.coverage.*
+import io.kotlintest.*
+import org.junit.jupiter.api.*
+
+class BuildMethodsTests : E2EPluginTest() {
+
+    @Test
+    fun `Deploy 2 builds without coverage`() {
+        createSimpleAppWithPlugin<CoverageSocketStreams> {
+            connectAgent<Build1> { plugUi, _ ->
+                plugUi.methods()!!.apply {
```

Comment:
why did u use '!!' ?

Reply:
to resolve the case when I get null in request and avoid assertion in apply brackets.